server/controllers/UserController.js

```diff
@@ -0,0 +1,106 @@
+import dotenv from 'dotenv';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import User from '../models/User';
+import { db } from '../database';
+
+dotenv.config();
+
+const secretKey = process.env.JWT_KEY;
+
+export default class UserController {
+  static createUser(req, res) {
+    // get user input
+    const {
+      firstName, lastName, email, password,
+    } = req.body;
+
+    // hash password
+    const passwordHash = bcrypt.hashSync(password, 10);
+
+    // create user object
+    const user = new User(firstName, lastName, email, passwordHash);
+
+    // save user to db
+    db.query(
+      'INSERT INTO users (first_name, last_name, email, role, password_hash, created_at) ' +
+      'VALUES ($1, $2, $3, $4, $5, $6) RETURNING *',
+      [user.firstName, user.lastName, user.email, user.role, user.password, 'NOW()'],
+    )
+      .then((result) => {
+        if (result.rowCount < 1) {
+          return res.status(500).json({
+            statusCode: 500,
+            error: 'Internal Server Error',
+            message: 'Oops! Another bug must have crawled into our systems, but we are right on it! Please try your request later :\'(',
```

Comment:
And since you are using the same message in multiple places, you can abstract this and create an helper function for it.